integration-box/csv_import_via_http/http_csv_download_sample.dig

```diff
@@ -0,0 +1,16 @@
+timezone: "Asia/Tokyo"
+
+schedule:
+ daily>: 07:45:00
+
++task1:
+  py>: csv_import_td.main
+  url           : CSV DOWNLOAD URL
```

Comment:
How about adding an example CSV in this repository and put the specific URL for it?

Reply:
Sounds nice! the url will be like `https://github.com/treasure-data/treasure-boxes/blob/master/integration-box/csv_import_via_http/sample_data.csv` , right?

Reply:
The URL should start `raw.githubusercontent.com` like:
`https://raw.githubusercontent.com/treasure-data/treasure-boxes/master/machine-learning-box/ml_tips/resources/titanic3.csv`